### PR TITLE
smbmap: 1.1.0 bump

### PR DESCRIPTION
**Pull request by gkroon**
Latest release moved to Python 3

### CODE REVIEW

File: net-analyzer/smbmap/smbmap-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+DESCRIPTION="SMBMap is a handy SMB enumeration tool"
+HOMEPAGE="https://github.com/ShawnDEvans/smbmap"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-python/impacket-0.9.20[${PYTHON_USEDEP}]
```
Comment (linxon) [inline net-analyzer/smbmap/smbmap-1.1.0.ebuild]: you need add py3.x support for `dev-python/impacket` too if it work fine with python3

File: net-analyzer/smbmap/smbmap-1.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	dev-python/pyasn1[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
+
+src_install() {
+	newbin "${PN}.py" "${PN}"
```
Comment (linxon) [inline net-analyzer/smbmap/smbmap-1.1.0.ebuild]: use 
```bash
python_foreach_impl python_newscript ${PN}.py ${PN}
```
instead newbin

File: net-analyzer/smbmap/smbmap-1.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+HOMEPAGE="https://github.com/ShawnDEvans/smbmap"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-python/impacket-0.9.20[${PYTHON_USEDEP}]
+	dev-python/pyasn1[${PYTHON_USEDEP}]
```
Comment (linxon) [inline net-analyzer/smbmap/smbmap-1.1.0.ebuild]: it also required `dev-python/pycrypto` https://github.com/ShawnDEvans/smbmap/blob/b667382f1bb12359a63ecf8f6374b417ef2ae2fc/requirements.txt#L1

add 
```bash
dev-python/pycrypto[${PYTHON_USEDEP}]
```
after this line too :)

File: net-analyzer/smbmap/smbmap-1.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit python-single-r1
```
Comment (linxon) [inline net-analyzer/smbmap/smbmap-1.1.0.ebuild]: `python-single-r1` can be replaced to `python-r1`

Comment (linxon) [reply net-analyzer/smbmap/smbmap-1.1.0.ebuild]: oh...

it seems this tool can write data files to a random directory
https://github.com/ShawnDEvans/smbmap/blob/b667382f1bb12359a63ecf8f6374b417ef2ae2fc/smbmap.py#L36-L50

maybe we need try to change it like this:
```diff
From 78b534e314590a1ed331393deaab9ef3aaeed07f Mon Sep 17 00:00:00 2001
From: Yury Martynov <email@linxon.ru>
Date: Fri, 1 Nov 2019 20:39:00 +0300
Subject: [PATCH] change workdir

---
 smbmap.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/smbmap.py b/smbmap.py
index 4db067c..a7792b6 100755
--- a/smbmap.py
+++ b/smbmap.py
@@ -33,11 +33,12 @@ import re
 # Seriously, the most amazing Python library ever!!
 # Many thanks to that dev team
 
-OUTPUT_FILENAME = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz',10))
-BATCH_FILENAME  = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10)) + '.bat'
-SMBSERVER_DIR   = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10))
+CURRENT_PATH = str(os.getcwd())
+OUTPUT_FILENAME = ''.join(CURRENT_PATH)
+BATCH_FILENAME  = ''.join(CURRENT_PATH) + '.bat'
+SMBSERVER_DIR   = ''.join(CURRENT_PATH)
 DUMMY_SHARE     = 'TMP'
-PERM_DIR = ''.join(random.sample('ABCDEFGHIGJLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz', 10))
+PERM_DIR = ''.join(CURRENT_PATH)
 
 class SMBServer(Thread):
     def __init__(self):
-- 
2.23.0


```

Comment (blshkv) [reply net-analyzer/smbmap/smbmap-1.1.0.ebuild]: Can't accept the patch. That would pollute the fs or fail to write a log if the current dir is read only.
It should write all output to ~/.smbmap or similar

**Pull request closed without merging**